redis/deprecated/redisbp/monitored_client.go

```diff
@@ -30,12 +37,47 @@ func NewMonitoredFailoverClient(name string, opt *redis.FailoverOptions) *redis.
 	return client
 }
 
+// ClusterClient extends redis cluster client with a functional Wait function
+type ClusterClient struct {
+	*redis.ClusterClient
+}
+
+// WaitArgs enclose inputs for Wait command into a struct
+type WaitArgs struct {
+	Key       string
+	NumSlaves int
```

Comment:
Unless you want to call this "ReplicationFactor", it probably doesn't make sense to use that term in the errors.

Reply:
also, `NumReplicas` might be a better name for this arg.